index.html

```diff
@@ -51,8 +59,12 @@ <h4>Skills</h4>
             </ul>
```

Comment:
- [ ] [_Optional_] Your skillset section does not seem to have the correct padding between its `li` elements as you can see in the images below. Also, the horizontal line is `white` compared to the Figma template which is `#646369`. Please do try to fix it whenever you can
> Yours
![image](https://user-images.githubusercontent.com/90556221/166121777-97e53295-f4bf-4250-8627-f8125f18f41f.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/166121793-f64ce084-e3d7-4798-92f0-d7894341b407.png)